proto/options.proto

```diff
@@ -166,6 +166,10 @@ message Options {
     invariant = 3;
   }
 
+  message Polygon {
+    repeated Location coords = 1;
```

Comment:
this is an important decision. the location object has with it all kinds of other options you can add. im pretty sure for this use-case we only ever just want the lat,lon but if we were to not use location we would need to do different parsing. anyway just something to think about

Reply:
hmm looks like you already have custom parsing. i would suggest changing this to LatLng object instead and not carry around all that baggage that comes with Location

Reply:
Call it `SimplePolygon` to leave the door open for more complex types?